spec/active_stash/consistency_spec.rb

```diff
@@ -11,32 +11,26 @@
   end
 
   describe "when the backing collection exists" do
-    before { User.collection.create! }
-    after { User.collection.drop! }
+    before(:example) { User.collection.create! }
+    after(:example) { User.collection.drop! }
 
     it "does not raise an error" do
       expect { User.collection.info }.to_not raise_error
     end
-  end
-
-  describe "when the backing collection exists but is missing an index" do
-    before { User.collection.create! }
-    after { User.collection.drop! }
 
-    it "raises an error" do
-      expect { UserInconsistent.collection(true).info }.to raise_error(ActiveStash::CollectionDivergedError)
-      UserInconsistent.collection.drop!
-      UserInconsistent.collection.create!
-      expect { UserInconsistent.collection(true).info }.to_not raise_error
+    describe "when the backing collection exists but is missing an index" do
```

Comment:
RSpec will automagically concatenate all the nested descriptions together to create the "full" description of a given test, so there's no need to duplicate "when the backing collection exists" in these subgroups.

Reply:
That's a copy pasta error - thanks!